tests/test_single_topology.py

```diff
@@ -272,6 +272,7 @@ def assert_chiral_atom_idxs_are_canonical(all_idxs):
         assert (j, k, l) < (k, l, j)
 
 
+@pytest.mark.nogpu
```

Comment:
This reduces the nightly tests total wall clock time by about an hour.

Reply:
Nightly pipeline: https://gitlab.com/relaytx/physics-platform/timemachine/-/pipelines/1182203931